### PR TITLE
adding only props to text element that it can recognize

### PR DESCRIPTION
**Pull request by ebrillhart**
removes the passing of props via {...props} and explicitly passes only attributes recognized by the `<text>` tag

addresses https://github.com/FormidableLabs/victory-chart/issues/281

@boygirl

### CODE REVIEW

File: src/victory-label/victory-label.js [hunk 1]
```diff
@@ -212,7 +212,12 @@ export default class VictoryLabel extends React.Component {
   renderElements(props, content) {
     const transform = this.getTransform(props);
     return (
-      <text {...props} transform={transform}>
+      <text dx={props.dx} dy={props.dy} x={props.x} y={props.y}
```
Comment (boygirl) [inline src/victory-label/victory-label.js]: This works fine, but take a look at this lodash method https://lodash.com/docs#pick

**Comment by ebrillhart** [comment]
@boygirl used pick but for some reason still had to list events separately as {...props.events} or else it was failing the test. let me know if there's a better way to handle this?

**Comment by boygirl** [comment]
@ebrillhart that's correct, because we actually want to spread the props object on to the element so that if `props.events = {onClick: onClickHandler}` we want the element to end up like `<text onClick={onClickHandler}/>` so that the events are actually attached.

**Comment by boygirl** [comment]
@ebrillhart Approved!

**Pull request merged**